src/pages/haeriRoh/Login/LoginHr.scss

```
@@ -0,0 +1,135 @@
+$fa-color: #fafafa;
+$wh-color: white;
+$focus-line: 1px solid gray;
+$border-style: 1px solid #dbdbdb;
+
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* css code */
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  width: 100%;
+  height: 100vh;
+  @include flex-center();
+  background-color: $fa-color;
```

```suggestion
  @include flex-center;
  background-color: $fa-color;
```

mixin에 받을 인수가 없다면 소괄호는 사용하지 않아도 됩니다!